src/main/java/xyz/vegaone/easytrackingv2/domain/BugEntity.java

```diff
@@ -0,0 +1,98 @@
+package xyz.vegaone.easytrackingv2.domain;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "bug")
+public class BugEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "description")
+    private String description;
+
+    @Column(name = "status")
+    private String status;
+
+    @Column(name = "priority")
+    private Integer priority;
+
+    @Column(name = "estimation")
+    private Integer estimation;
+
+    @Column(name = "severity")
+    private String severity;
+
+    @ManyToOne
+    @JoinColumn(name = "user_story_id")
+    private UserStoryEntity userStory;
+
```

Comment:
A bug also has an owner. A user that is responsible for it. Please add one. Should be a many to one relationship.